examples/java/.gitignore

```diff
@@ -1,25 +1,83 @@
-### Gradle stuff
+### https://raw.github.com/github/gitignore/da00310ccba9de9a988cc973ef5238ad2c1460e9/java.gitignore
```

Comment:
this is a gitignore file taken from ratpack which already ignores the right things for gradle